torch/export/_trace.py

```diff
@@ -644,7 +644,7 @@ def make_argument_spec(i, node) -> ArgumentSpec:
     )
 
     @dataclasses.dataclass
-    class _ExportedProgramNonStrict:
+    class _ATenIRExportedProgram:
```

Comment:
nit: sorry could we rename this to something like, `_ExportedArtifact`, since it's not really an exported program?